src/components/oscal-utils/OSCALLinkUtils.js

```diff
@@ -58,3 +58,76 @@ export function guessExtensionFromHref(url) {
   }
   return detectedExtension;
 }
+
+/**
+ * Finds if a fragment contains a control group by searching the control tabs
+ *
+ * @param {string} fragment A given fragment
+ * @returns {string} The control group id
+ */
+export function determineControlGroupFromFragment(fragment) {
+  if (fragment === null || fragment === "" || fragment === undefined) {
+    return null;
+  }
+  // Create array from all tab control grouping elements
+  const controlGroupList = Array.from(
+    document.querySelectorAll('[id^="vertical-tab-"]')
+  );
+  // Grab group id if found
+  return controlGroupList
+    ?.find(
+      (group) =>
+        fragment.split("/")?.[0] === group.id.split("vertical-tab-").pop()
+    )
+    ?.id?.split("vertical-tab-")
+    ?.pop();
+}
+
+/**
+ * Add (push) a control/sub-control to fragmentPrefix.
+ *
+ * @param {string} fragmentPrefix The beginning of a fragment
+ * @param {string} controlId The identification for a control
+ * @returns {string} fragmentPrefix with an added control
+ */
+export function appendToFragmentPrefix(fragmentPrefix, controlId) {
+  return fragmentPrefix && fragmentPrefix !== ""
+    ? `${fragmentPrefix}/${controlId}`
+    : controlId;
```

Comment:
Same thing here. I am pretty sure `""` is falsy. 

```suggestion
  return fragmentPrefix
    ? `${fragmentPrefix}/${controlId}`
    : controlId;
```